Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -424,17 +421,26 @@ public class Fuzzer {
         }
         // Save result of runtime types collection to Program
         program.typeCollectionStatus = TypeCollectionStatus(from: execution.outcome)
+    }
 
-        // Recollect types using AbstractInterpreter as we have new precise runtime type hints
-        inferMissingTypes(in: program)
+    func computeTypeInformation(for program: Program, minimized: Bool) {
+        if config.collectRuntimeTypes {
+            collectRuntimeTypes(for: program)
+        }
+        // New static types are needed either if program was minimized and lost its type info
+        // or we collected runtime types and statical inference can be improved now
+        let newTypesNeeded = minimized || config.collectRuntimeTypes
```

Comment:
Could we make a `Program.typeInformationAvailable` property/method or a `program.types.isEmpty` or so, and then replace `minimized` with that in this condition? I think that would be a bit more intuitive, since I guess there could be other reasons in the future why a program doesn't have type information.